### PR TITLE
Explore base improvements

### PR DESCRIPTION
**Pull request by hmvp**
Some small improvements found when using the explorer base bsp

### CODE REVIEW

**Comment by 9names** [comment]
The PIO driver deps are getting bumped to released versions in #362 , no need to switch them to the main branch.

**Comment by hmvp** [comment]
> The PIO driver deps are getting bumped to released versions in #362 , no need to switch them to the main branch.

Yeah I didn't want to, but the build is currently broken because the hal_0.5.0 branch does not exist anymore.. So either we merge this quickly or ill drop the commit when #362 is merged...

**Comment by 9names** [comment]
#362 is merged, apologies for the breakage

**Review by 9names — APPROVED** [review]
LGTM

File: rp2040-hal/src/timer.rs [hunk 1]
```diff
@@ -288,27 +288,27 @@ pub enum ScheduleAlarmError {
 impl_alarm!(Alarm0 {
     rb: alarm0,
     int: alarm_0,
-    int_name: "IRQ_TIMER_0",
+    int_name: "TIMER_IRQ_0",
```
[the remark this answers is not in the document]
Comment (jannic) [reply rp2040-hal/src/timer.rs:291]: This value is only used in the doc comments, so the impact on HAL users is minimal.

**Review by jannic — APPROVED** [review]

**Pull request merged**